dotcom-rendering/src/web/components/GetMatchNav.importable.tsx

```diff
@@ -17,12 +17,12 @@ export const GetMatchNav = ({ matchUrl }: Props) => {
 		comments?: string;
 		minByMinUrl?: string;
 		venue?: string;
-	}>(matchUrl);
+	}>(matchUrl, { errorRetryCount: 1 });
```

Comment:
If the match stats exist, but we error randomly once, the match stats will be removed from the page? Would the reader have to refresh the page to then see match stats again (given the response returns what's expected)?

Only flagging this in case we think an error retry count of 1 is too strict, but I have no idea of the error rate in production.

Reply:
You’d have to error twice in a row (1 attempt + 1 retry). This sounded sane.